packages/genesys-spark-components/src/components/beta/gux-tooltip-base/gux-tooltip-base.tsx

```diff
@@ -230,6 +246,7 @@ export class GuxTooltipBase {
           data-placement={this.placement}
         >
           <slot name="content" />
+          <div id="arrow"></div>
```

Comment:
If you use something like
```
<div class="arrow" ref={el => (this. arrowElement = el)}></div>
```
you won't need to manually find the element in the setForElement function